File: avancado-intermediario/Python/Q4/universidade_fisica.py
from universidade import Universidade

class UniversidadeFisica(Universidade):

    def __init__(self, nome, campus):
        super().__init__(nome)
        self.__campus = campus

    def getCampus(self):
        return self.__campus

    def setCampus(self, campus):
        self.__campus = campus

    def aplicarAvaliacao(self):
        print(f"Avaliações presenciais no campus {self.getCampus()}")

    def __str__(self) -> str:
        return f"UniversidadeFisica: {self.getNome()} | Campus: {self.getCampus()}"

File: avancado-intermediario/Java/Q4/UniversidadeFisica.java
public class UniversidadeFisica extends Universidade{
    private String campus;

    public UniversidadeFisica(String nome, String campus){
        super(nome);
        this.campus = campus;
    }

    public String getCampus(){
        return this.campus;
    }

    public void setCampus(String campus){
        this.campus = campus;
    }

    public void aplicarAvaliacao(){
        System.out.println("Avaliações presenciais no campus " + getCampus());
    }

    @Override
    public String toString() {
        return super.toString() + " (Física) | Campus: " + campus;
    }
}